classes/ClaytonTestClassTest.cls

```diff
@@ -0,0 +1,8 @@
+@isTest(SeeAllData=true)
+private class ClaytonTestClassTest {
+    private testmethod void testSuite(){
+		System.debug('This is not a test');
+		Account acc = [Select Id, Name From Account];
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. Consider either introducing a `WHERE` clause, or adding a `LIMIT` to contain the number of records returned.